types/wordpress__blocks/api/registration.d.ts

```diff
@@ -167,3 +176,29 @@ export function unregisterBlockStyle(blockName: string, styleVariationName: stri
  * @returns The previous block value if successfully unregistered, otherwise `undefined`.
  */
 export function unregisterBlockType(name: string): Block<any> | undefined;
+
+/**
+ * Returns an array with the variations of a given block type.
+ *
+ * @param blockName - Name of block (example: “core/columns”).
+ * @param [scope] - Block variation scope name.
+ *
+ * @returns Block variations.
+ */
+export function getBlockVariations(blockName: string, scope?: BlockVariationScope): BlockVariation[] | undefined;
+
+/**
+ * Registers one or more new block variations for the given block type.
+ *
+ * @param blockName - Name of the block (example: “core/columns”).
+ * @param variation - Object or array of objects describing block variations.
```

Comment:
Nitpick: feel free to ignore, but I find the wording here awkward, as if I should be more focused on "Object" than "variation." I wonder if a rephrasing could clarify:

```ts
 * @param variation - Variation description object (or array if more than one).
```